scan.conf

```diff
@@ -0,0 +1,793 @@
+##
+## Example config file for the Clam AV daemon
```

Comment:
I think this config is really better to be a part of the https://github.com/WikiTeq/docker-centre.santafe.edu repo since it implies that the Docker host has the `clamd` installed and otherwise the `clamscan` won't work. This is very specific for one site's needs and should not be part of the base image.

This can be ignored if you're out of time on the task though, but let's create a new issue to fix this later in that case

Reply:
It is clamav client only, it sends files to another host for checking.   so, I think it can be a part of the docker image

Reply:
fine, we can handle & document it later